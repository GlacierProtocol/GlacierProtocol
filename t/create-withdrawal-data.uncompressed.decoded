{
  "txid": "31906179a4f3d983f62cecd9e8fcd4bc3df4969e3d5ec85547b3fecbdb53c51c",
  "hash": "31906179a4f3d983f62cecd9e8fcd4bc3df4969e3d5ec85547b3fecbdb53c51c",
  "version": 2,
  "size": 534,
  "vsize": 534,
  "weight": 2136,
  "locktime": 0,
  "vin": [
    {
      "txid": "2ddaa2de6e72813c5783274f2408d5ebcab33dedaf03ec38ea1243c67108a19f",
      "vout": 1,
      "scriptSig": {
        "asm": "0 304402206a3d176d78280fedc2b8106ac1c2a28b1d67c2ef671646be403ac2abe48d709b02203cad17a7ae5b0773a141af4eb0160e4fa6404ba151019437b683a8bc11fbcca4[ALL] 3044022066e85dd73c2fc0b6478f3e7b1c1f0582abdc05efaa48f24cc11cbc788c64a31502207b403dbe2efa6227d20898f79ea86c696887252f29a63cafc779004fb649777a[ALL] 52410477f15f22aeffaf3f3bc48a280a767f7c6af21276783c09ff3bcaabbece1781135cf0f28f643d3c60a75c29db0819d5442bae83d8f74e7c44ced3255665998e52410403f642223bc751e125ab604ed20c544e933eb82a19201f90bda6236a9ddbedac931c108c7c3189b7fc7ea7097b02217d819e866a9841291c522f69173a12524a41046895f3ed01c09025ba9cfab7b67d699e3ebaceada0beff70bc72c7f10df2d1c674df743bbb58cea21170831334e4d6d965ed34ce23249557091f440305ab9c0041049f3cb152130c241e01b9220a147f69f7111b830def35b510973ab47bb9af7ff6f21fcb5a958839fee0022e8ea5ddc6d71fc624a84bdc42835193abfee125b1c354ae",
        "hex": "0047304402206a3d176d78280fedc2b8106ac1c2a28b1d67c2ef671646be403ac2abe48d709b02203cad17a7ae5b0773a141af4eb0160e4fa6404ba151019437b683a8bc11fbcca401473044022066e85dd73c2fc0b6478f3e7b1c1f0582abdc05efaa48f24cc11cbc788c64a31502207b403dbe2efa6227d20898f79ea86c696887252f29a63cafc779004fb649777a014d0b0152410477f15f22aeffaf3f3bc48a280a767f7c6af21276783c09ff3bcaabbece1781135cf0f28f643d3c60a75c29db0819d5442bae83d8f74e7c44ced3255665998e52410403f642223bc751e125ab604ed20c544e933eb82a19201f90bda6236a9ddbedac931c108c7c3189b7fc7ea7097b02217d819e866a9841291c522f69173a12524a41046895f3ed01c09025ba9cfab7b67d699e3ebaceada0beff70bc72c7f10df2d1c674df743bbb58cea21170831334e4d6d965ed34ce23249557091f440305ab9c0041049f3cb152130c241e01b9220a147f69f7111b830def35b510973ab47bb9af7ff6f21fcb5a958839fee0022e8ea5ddc6d71fc624a84bdc42835193abfee125b1c354ae"
      },
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 0.00040660,
      "n": 0,
      "scriptPubKey": {
        "asm": "OP_HASH160 762fea3246dc4ce19cfd3f1aa9685e8838605057 OP_EQUAL",
        "hex": "a914762fea3246dc4ce19cfd3f1aa9685e883860505787",
        "reqSigs": 1,
        "type": "scripthash",
        "addresses": [
          "2N42986nuqGfmbRUuojYCboE7EE9VLmYai2"
        ]
      }
    },
    {
      "value": 0.00004000,
      "n": 1,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 fb5b6788a336c0cf49b90e51ad426f2e89bf43fb OP_EQUALVERIFY OP_CHECKSIG",
        "hex": "76a914fb5b6788a336c0cf49b90e51ad426f2e89bf43fb88ac",
        "reqSigs": 1,
        "type": "pubkeyhash",
        "addresses": [
          "n4S1Q6S9b5dq72ttpnFZXwaPCs2Z8ojwPR"
        ]
      }
    }
  ]
}
