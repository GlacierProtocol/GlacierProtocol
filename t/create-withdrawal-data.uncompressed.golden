Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
You will need to enter several pieces of information to create a withdrawal transaction.


*** PLEASE BE SURE TO ENTER THE CORRECT DESTINATION ADDRESS ***


Source cold storage address: 
Redemption script for source cold storage address: 
Destination address: 
How many unspent transactions will you be using for this withdrawal? 
Please paste raw transaction #1 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Transaction data found for source address.
TOTAL unspent amount for this raw transaction: 0.00050000 BTC

How many private keys will you be signing this transaction with? 
#: Key #1: Key #2: 
Enter fee rate.
Satoshis per vbyte: Calculated fee (0.05340000) is too high. Must be under 0.005

Enter fee rate.
Satoshis per vbyte: 
Based on the provided rate, the fee will be 0.00005340 bitcoin.
Confirm? (y/n): 
Please enter the decimal amount (in bitcoin) to withdraw to the destination address.

Example: For 2.3 bitcoins, enter "2.3".

After a fee of 0.00005340, you have 0.00044660 bitcoins available to withdraw.

*** Technical note for experienced Bitcoin users:  If the withdrawal amount & fee are cumulatively less than the total amount of the unspent transactions, the remainder will be sent back to the same cold storage address as change. ***

Amount to send to n4S1Q6S9b5dq72ttpnFZXwaPCs2Z8ojwPR (leave blank to withdraw all funds stored in these unspent transactions): 0.00040660 being returned to cold storage address address 2N42986nuqGfmbRUuojYCboE7EE9VLmYai2.

Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.00050000 BTC in unspent supplied transactions
0.00040660 BTC going back to cold storage address 2N42986nuqGfmbRUuojYCboE7EE9VLmYai2
0.00004000 BTC going to destination address n4S1Q6S9b5dq72ttpnFZXwaPCs2Z8ojwPR
Fee amount: 0.00005340

Signing with private keys: 
92jodMpEwGtS11sMjxfq6acnc1Yi6VX2QFZWr4YQVjXnQGdKH1g
922F7weRdtBgCnqcnuMo8hba8wj75fZc2nx4oTnLHevh9GaCVXA


Confirm? (y/n): 
Calculating transaction...


Sufficient private keys to execute transaction?
True

Raw signed transaction (hex):
02000000019fa10871c64312ea38ec03afed3db3caebd508244f2783573c81726edea2da2d01000000fd9f010047304402206a3d176d78280fedc2b8106ac1c2a28b1d67c2ef671646be403ac2abe48d709b02203cad17a7ae5b0773a141af4eb0160e4fa6404ba151019437b683a8bc11fbcca401473044022066e85dd73c2fc0b6478f3e7b1c1f0582abdc05efaa48f24cc11cbc788c64a31502207b403dbe2efa6227d20898f79ea86c696887252f29a63cafc779004fb649777a014d0b0152410477f15f22aeffaf3f3bc48a280a767f7c6af21276783c09ff3bcaabbece1781135cf0f28f643d3c60a75c29db0819d5442bae83d8f74e7c44ced3255665998e52410403f642223bc751e125ab604ed20c544e933eb82a19201f90bda6236a9ddbedac931c108c7c3189b7fc7ea7097b02217d819e866a9841291c522f69173a12524a41046895f3ed01c09025ba9cfab7b67d699e3ebaceada0beff70bc72c7f10df2d1c674df743bbb58cea21170831334e4d6d965ed34ce23249557091f440305ab9c0041049f3cb152130c241e01b9220a147f69f7111b830def35b510973ab47bb9af7ff6f21fcb5a958839fee0022e8ea5ddc6d71fc624a84bdc42835193abfee125b1c354aeffffffff02d49e00000000000017a914762fea3246dc4ce19cfd3f1aa9685e883860505787a00f0000000000001976a914fb5b6788a336c0cf49b90e51ad426f2e89bf43fb88ac00000000

Transaction fingerprint (md5):
3ed21f029417316aa811b14f062244dc
QR code for transaction written to transaction.png
